diff -Naurp fwlogwatch-1.1/contrib/fwlogsummary.cgi fwlogwatch-1.1.oden/contrib/fwlogsummary.cgi
--- fwlogwatch-1.1/contrib/fwlogsummary.cgi	2006-03-08 20:36:08.000000000 +0100
+++ fwlogwatch-1.1.oden/contrib/fwlogsummary.cgi	2008-05-12 15:17:07.000000000 +0200
@@ -18,7 +18,7 @@ date
 
 RECENT="-l 1h"
 WEBDIR="/var/www/html/fwlogwatch"
-FWLOGWATCH="/usr/local/sbin/fwlogwatch"
+FWLOGWATCH="/usr/sbin/fwlogwatch"
 
 if [ ! -d $WEBDIR ] ; then
   echo "Directory $WEBDIR does not exist!"
diff -Naurp fwlogwatch-1.1/contrib/fwlogsummary_small.cgi fwlogwatch-1.1.oden/contrib/fwlogsummary_small.cgi
--- fwlogwatch-1.1/contrib/fwlogsummary_small.cgi	2006-03-08 20:36:08.000000000 +0100
+++ fwlogwatch-1.1.oden/contrib/fwlogsummary_small.cgi	2008-05-12 15:17:07.000000000 +0200
@@ -4,4 +4,4 @@
 
 echo "Content-Type: text/html"
 echo
-/usr/local/sbin/fwlogwatch -w -l 1h -z -s -d
+/usr/sbin/fwlogwatch -w -l 1h -z -s -d
diff -Naurp fwlogwatch-1.1/contrib/fwlogwatch.php fwlogwatch-1.1.oden/contrib/fwlogwatch.php
--- fwlogwatch-1.1/contrib/fwlogwatch.php	2006-03-08 20:36:08.000000000 +0100
+++ fwlogwatch-1.1.oden/contrib/fwlogwatch.php	2008-05-12 15:17:07.000000000 +0200
@@ -16,7 +16,7 @@
 $debug = '0';
 $logdir = "/var/log";
 $logbase = messages;
-$fwlogwatch = "/usr/local/sbin/fwlogwatch";
+$fwlogwatch = "/usr/sbin/fwlogwatch";
 
 /**********************************************************************/
 
diff -Naurp fwlogwatch-1.1/fwlogwatch.config fwlogwatch-1.1.oden/fwlogwatch.config
--- fwlogwatch-1.1/fwlogwatch.config	2004-03-23 14:09:21.000000000 +0100
+++ fwlogwatch-1.1.oden/fwlogwatch.config	2008-05-12 15:19:29.000000000 +0200
@@ -328,8 +328,8 @@
 # Alternative paths for the notification and response scripts can be
 # specified with the 'notification_script' and 'response_script' options.
 #
-#notification_script = /usr/local/sbin/fwlw_notify
-#response_script = /usr/local/sbin/fwlw_respond
+#notification_script = /usr/sbin/fwlw_notify
+#response_script = /usr/sbin/fwlw_respond
 
 # Known hosts are those that will not be warned about or actions taken
 # against, even if they match the alert/response criteria.
@@ -356,7 +356,7 @@
 # Command line option: -X <port>
 #
 #server_status = no
-#bind_to = 127.0.0.1
+bind_to = 127.0.0.1
 #listen_port = 888
 #listen_to =
 #status_user = admin
diff -Naurp fwlogwatch-1.1/main.h fwlogwatch-1.1.oden/main.h
--- fwlogwatch-1.1/main.h	2006-04-17 16:54:51.000000000 +0200
+++ fwlogwatch-1.1.oden/main.h	2008-05-12 15:17:07.000000000 +0200
@@ -10,16 +10,14 @@
 
 /* Paths */
 
-#define INSTALL_DIR "/usr/local"
 #define CONF_DIR "/etc"
-#define LOCALE_DIR "/usr"
 
 /* i18n */
 
 #ifdef HAVE_GETTEXT
 #include <libintl.h>
 #define _(String) gettext(String)
-#define LOCALEDIR LOCALE_DIR "/share/locale"
+#define LOCALEDIR "/usr/share/locale"
 #else
 #define _(String) String
 #endif
@@ -309,8 +307,8 @@ enum {
 
 #define ALERT 5
 #define FORGET 86400
-#define FWLW_NOTIFY INSTALL_DIR "/sbin/fwlw_notify"
-#define FWLW_RESPOND INSTALL_DIR "/sbin/fwlw_respond"
+#define FWLW_NOTIFY "/usr/sbin/fwlw_notify"
+#define FWLW_RESPOND "/usr/sbin/fwlw_respond"
 #define STATUS_TITLE _("fwlogwatch status")
 #ifndef HAVE_IPV6
 #define LISTENIF "127.0.0.1"
diff -Naurp fwlogwatch-1.1/Makefile fwlogwatch-1.1.oden/Makefile
--- fwlogwatch-1.1/Makefile	2006-03-08 20:36:02.000000000 +0100
+++ fwlogwatch-1.1.oden/Makefile	2008-05-12 15:17:07.000000000 +0200
@@ -10,41 +10,24 @@
 
 # Linux
 CC = gcc
-CFLAGS = -DHAVE_ZLIB -DHAVE_GETTEXT -pipe -O2 -Wall #-pedantic -Wpointer-arith #-g #-p
+CFLAGS = $(OPTFLAGS) -DHAVE_ZLIB -DHAVE_GETTEXT -DHAVE_IPV6 -DHAVE_ADNS
 LDFLAGS = #-g #-static -p
-LIBS = -lcrypt -lz #-ladns #-lc_p
-
-# Solaris
-#LIBS = -lnsl -lsocket -lcrypt -lz
-#
-#CC = gcc
-#CFLAGS = -DSOLARIS -DHAVE_ZLIB -DHAVE_GETTEXT -pipe -O2 -Wall #-pedantic #-g
-#LDFLAGS = #-g
-#
-#CC = cc
-#CFLAGS = -DSOLARIS -DHAVE_ZLIB -DHAVE_GETTEXT -v -fast -xCC
-
-# OpenBSD
-#CC = gcc
-#CFLAGS = -DHAVE_ZLIB -DHAVE_GETTEXT -pipe -O2 -Wall -I/usr/local/include
-#LIBS = -L/usr/local/lib -lz -lintl
-
-# FreeBSD
-#CC = gcc
-#CFLAGS = -DHAVE_ZLIB -DHAVE_GETTEXT -pipe -O2 -Wall -I/usr/local/include
-#LIBS = -L/usr/local/lib -lcrypt -lz -lintl
-
+LIBS = -lcrypt -lz -ladns
 
 LEX = flex
 LFLAGS = -B --nounput #-f #-p -p -d
 
 INSTALL = install
-INSTALL_PROGRAM = $(INSTALL) -s -m 0755
+INSTALL_PROGRAM = $(INSTALL) -m 0755
 INSTALL_SCRIPT = $(INSTALL) -m 0755
 INSTALL_DATA = $(INSTALL) -m 0644
-INSTALL_DIR = /usr/local
-CONF_DIR = /etc
-LOCALE_DIR = /usr
+INSTALL_DIR = $(INSTALL) -d -m 0755
+
+CONF_DIR=/etc
+LOCALE_DIR=/usr/share/locale
+SBIN_DIR=/usr/sbin
+MAN_DIR=/usr/share/man
+DESTDIR=
 
 OBJS = cisco_ios.o cisco_pix.o compare.o ipchains.o ipfilter.o ipfw.o \
        lancom.o main.o modes.o net.o netfilter.o netscreen.o output.o \
@@ -79,38 +62,48 @@
 	$(CC) $(LDFLAGS) -o $@ $(OBJS) $(LIBS)
 
 install:	all
-	$(INSTALL_PROGRAM) fwlogwatch $(INSTALL_DIR)/sbin/fwlogwatch
-	$(INSTALL_SCRIPT) contrib/fwlw_notify $(INSTALL_DIR)/sbin/fwlw_notify
-	$(INSTALL_SCRIPT) contrib/fwlw_respond $(INSTALL_DIR)/sbin/fwlw_respond
-	$(INSTALL_DATA) fwlogwatch.8 $(INSTALL_DIR)/share/man/man8/fwlogwatch.8
-
+	$(INSTALL_DIR) $(DESTDIR)$(SBIN_DIR)
+	$(INSTALL_DIR) $(DESTDIR)$(MAN_DIR)/man8
+	$(INSTALL_PROGRAM) fwlogwatch $(DESTDIR)$(SBIN_DIR)/fwlogwatch
+	$(INSTALL_SCRIPT) contrib/fwlw_notify $(DESTDIR)$(SBIN_DIR)/fwlw_notify
+	$(INSTALL_SCRIPT) contrib/fwlw_respond $(DESTDIR)$(SBIN_DIR)/fwlw_respond
+	$(INSTALL_DATA) fwlogwatch.8 $(DESTDIR)$(MAN_DIR)/man8/fwlogwatch.8
 install-config:
-	$(INSTALL_DATA) fwlogwatch.config $(CONF_DIR)/fwlogwatch.config
+	$(INSTALL_DIR) $(DESTDIR)$(CONF_DIR)
+	$(INSTALL_DATA) fwlogwatch.config $(DESTDIR)$(CONF_DIR)/fwlogwatch.config
+	$(INSTALL_DATA) fwlogwatch.template $(DESTDIR)$(CONF_DIR)/fwlogwatch.template
 
 install-i18n:
 	cd po; make
-	$(INSTALL_DATA) po/de.mo $(LOCALE_DIR)/share/locale/de/LC_MESSAGES/fwlogwatch.mo
-	$(INSTALL_DATA) po/ja.mo $(LOCALE_DIR)/share/locale/ja/LC_MESSAGES/fwlogwatch.mo
-	$(INSTALL_DATA) po/pt.mo $(LOCALE_DIR)/share/locale/pt/LC_MESSAGES/fwlogwatch.mo
-	$(INSTALL_DATA) po/sv.mo $(LOCALE_DIR)/share/locale/sv/LC_MESSAGES/fwlogwatch.mo
-	$(INSTALL_DATA) po/zh_CN.mo $(LOCALE_DIR)/share/locale/zh_CN/LC_MESSAGES/fwlogwatch.mo
-	$(INSTALL_DATA) po/zh_TW.mo $(LOCALE_DIR)/share/locale/zh_TW/LC_MESSAGES/fwlogwatch.mo
+	$(INSTALL_DIR) $(DESTDIR)$(LOCALE_DIR)/de/LC_MESSAGES
+	$(INSTALL_DIR) $(DESTDIR)$(LOCALE_DIR)/ja/LC_MESSAGES
+	$(INSTALL_DIR) $(DESTDIR)$(LOCALE_DIR)/pt/LC_MESSAGES
+	$(INSTALL_DIR) $(DESTDIR)$(LOCALE_DIR)/sv/LC_MESSAGES
+	$(INSTALL_DIR) $(DESTDIR)$(LOCALE_DIR)/zh_CN/LC_MESSAGES
+	$(INSTALL_DIR) $(DESTDIR)$(LOCALE_DIR)/zh_TW/LC_MESSAGES
+	$(INSTALL_DATA) po/de.mo $(DESTDIR)$(LOCALE_DIR)/de/LC_MESSAGES/fwlogwatch.mo
+	$(INSTALL_DATA) po/ja.mo $(DESTDIR)$(LOCALE_DIR)/ja/LC_MESSAGES/fwlogwatch.mo
+	$(INSTALL_DATA) po/pt.mo $(DESTDIR)$(LOCALE_DIR)/pt/LC_MESSAGES/fwlogwatch.mo
+	$(INSTALL_DATA) po/sv.mo $(DESTDIR)$(LOCALE_DIR)/sv/LC_MESSAGES/fwlogwatch.mo
+	$(INSTALL_DATA) po/zh_CN.mo $(DESTDIR)$(LOCALE_DIR)/zh_CN/LC_MESSAGES/fwlogwatch.mo
+	$(INSTALL_DATA) po/zh_TW.mo $(DESTDIR)$(LOCALE_DIR)/zh_TW/LC_MESSAGES/fwlogwatch.mo
 
 install-rhinit:
-	$(INSTALL_SCRIPT) contrib/fwlogwatch.init.redhat $(CONF_DIR)/rc.d/init.d/fwlogwatch
+	$(INSTALL_DIR) $(DESTDIR)$(CONF_DIR)
+	$(INSTALL_SCRIPT) contrib/fwlogwatch.init.redhat $(DESTDIR)$(CONF_DIR)/rc.d/init.d/fwlogwatch
 
 uninstall:
-	@rm -f $(INSTALL_DIR)/sbin/fwlogwatch \
-		$(INSTALL_DIR)/sbin/fwlw_notify \
-		$(INSTALL_DIR)/sbin/fwlw_respond \
-		$(INSTALL_DIR)/share/man/man8/fwlogwatch.8 \
-		$(LOCALE_DIR)/share/locale/de/LC_MESSAGES/fwlogwatch.mo \
-		$(LOCALE_DIR)/share/locale/ja/LC_MESSAGES/fwlogwatch.mo \
-		$(LOCALE_DIR)/share/locale/pt/LC_MESSAGES/fwlogwatch.mo \
-		$(LOCALE_DIR)/share/locale/sv/LC_MESSAGES/fwlogwatch.mo \
-		$(LOCALE_DIR)/share/locale/zh_CN/LC_MESSAGES/fwlogwatch.mo \
-		$(LOCALE_DIR)/share/locale/zh_TW/LC_MESSAGES/fwlogwatch.mo \
-		$(CONF_DIR)/fwlogwatch.config \
+	@rm -f $(DESTDIR)/$(SBIN_DIR)fwlogwatch \
+		$(DESTDIR)$(SBIN_DIR)/fwlw_notify \
+		$(DESTDIR)$(SBIN_DIR)/fwlw_respond \
+		$(DESTDIR)$(MAN_DIR)/man8/fwlogwatch.8 \
+		$(DESTDIR)$(LOCALE_DIR)/de/LC_MESSAGES/fwlogwatch.mo \
+		$(DESTDIR)$(LOCALE_DIR)/ja/LC_MESSAGES/fwlogwatch.mo \
+		$(DESTDIR)$(LOCALE_DIR)/pt/LC_MESSAGES/fwlogwatch.mo \
+		$(DESTDIR)$(LOCALE_DIR)/sv/LC_MESSAGES/fwlogwatch.mo \
+		$(DESTDIR)$(LOCALE_DIR)/zh_CN/LC_MESSAGES/fwlogwatch.mo \
+		$(DESTDIR)$(LOCALE_DIR)/zh_TW/LC_MESSAGES/fwlogwatch.mo \
+		$(DESTDIR)$(CONF_DIR)/fwlogwatch.config
 
 clean:
 	rm -f *.o *~ *.bak fwlogwatch
diff -Naurp fwlogwatch-1.1/README fwlogwatch-1.1.oden/README
--- fwlogwatch-1.1/README	2005-03-15 12:36:43.000000000 +0100
+++ fwlogwatch-1.1.oden/README	2008-05-12 15:17:07.000000000 +0200
@@ -99,7 +99,7 @@
   read access to the system's standard log file (e.g. /var/log/messages)
   you can use group permissions. You can also bind a nonprivileged port for
   the status server. 'make install' will install the binary (in
-  /usr/local/sbin) and the man page, 'make install-config' will also
+  /usr/sbin) and the man page, 'make install-config' will also
   install a sample configuration file in /etc.
 
 - Zlib/Gettext/IPv6/GNU adns
